src/main/resources/application.yml

```
@@ -1,8 +1,17 @@
 spring:
+  h2:
+    console:
+      enabled: true
+      path: '/h2-console'
   liquibase:
     change-log: classpath:/db/changelog.yml
```

```suggestion
    change-log: classpath:/db/changelog.yml
    contexts: prod
```

Add liquibase context to not add test data on real environments.
We should discuss whether we will put this on in application.yml or application-cloud.yml. Maybe the testdata is usefull for local execution.